streams/src/test/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyKeyValueStoreTest.java

```diff
@@ -171,6 +208,10 @@ public void shouldThrowInvalidStoreExceptionDuringRebalance() throws Exception {
         rebalancing().get("anything");
     }
 
+    @Test(expected = InvalidStateStoreException.class)
+    public void shouldThrowInvalidStoreExceptionOnApproximateNumEntriesDuringRebalance() throws Exception {
+        rebalancing().approximateNumEntries();
```

Comment:
Can `rebalancing()` throw a `InvalidStateStoreException` ? If yes, we need to split this an apply try-catch-fail pattern instead of using `@expected`